core/src/main/scala/cats/Traverse.scala

```diff
@@ -213,6 +213,22 @@ trait Traverse[F[_]] extends Functor[F] with Foldable[F] with UnorderedTraverse[
 
   override def unorderedSequence[G[_]: CommutativeApplicative, A](fga: F[G[A]]): G[F[A]] =
     sequence(fga)
+
+  def foldTraverse[A, B, G[_]](fa: F[A])(f: A => G[B])(implicit G: Monad[G], M: Monoid[B]): G[B] =
+    foldM(fa, M.empty) { case (acc, a) =>
+      G.map(f(a)) { b =>
+        M.combine(acc, b)
+      }
+    }
+
+  def foldTraverseK[A, B, G[_], H[_]](fa: F[A])(f: A => G[H[B]])(implicit G: Monad[G], M: MonoidK[H]): G[H[B]] =
+    foldTraverse(fa)(f)(G, M.algebra)
+
+  def foldSequence[A, G[_]](fga: F[G[A]])(implicit G: Monad[G], M: Monoid[A]): G[A] =
```

Comment:
This is `foldA`.

https://github.com/typelevel/cats/blob/14286c9ef89a652fcff182bbd685f6caf930701e/core/src/main/scala/cats/Foldable.scala#L500-L501